### PR TITLE
add dp test help doc

### PR DESCRIPTION
**Pull request by likefallwind**
add dp test help doc

### CODE REVIEW

File: deepmd/entrypoints/main.py [hunk 1]
```diff
@@ -281,7 +281,7 @@ def main_parser() -> argparse.ArgumentParser:
         "--detail-file",
         type=str,
         default=None,
-        help="File where details of energy force and virial accuracy will be written",
+        help="Files where details of energy, force and virial accuracy/accuracy per atom will be written",
```
Comment (njzjz) [inline deepmd/entrypoints/main.py:284]: Actually the argument assigns **the prefix of** files.

Comment (likefallwind) [reply deepmd/entrypoints/main.py:284]: So maybe the proper description is "The prefix to files where details of energy, force and virial accuracy/accuracy per atom will be written"?

Comment (njzjz) [reply deepmd/entrypoints/main.py:284]: Agree

Comment (likefallwind) [reply deepmd/entrypoints/main.py]: ok I have changed the file

**Review by njzjz — APPROVED** [review]

**Pull request merged**